packages/block-editor/src/components/block-parent-selector/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * WordPress dependencies
+ */
+import { getBlockType } from '@wordpress/blocks';
+import { Button } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import BlockIcon from '../block-icon';
+
+/**
+ * Block parent selector component, displaying the hierarchy of the
+ * current block selection as a single icon to "go up" a level.
+ *
+ * @return {WPElement} Parent Block Selector.
+ */
+export default function BlockParentSelector() {
+	const { selectBlock } = useDispatch( 'core/block-editor' );
+	const { parents } = useSelect( ( select ) => {
+		const { getSelectedBlockClientId, getBlockParents } = select(
+			'core/block-editor'
+		);
+		const selectedBlockClientId = getSelectedBlockClientId();
+		return {
+			parents: getBlockParents( selectedBlockClientId ),
+		};
+	}, [] );
```

Comment:
Since we're only using `useSelect` for a single value, we can just return it and avoid having to destructure:
```suggestion
	const parents = useSelect( ( select ) => {
		const { getSelectedBlockClientId, getBlockParents } = select(
			'core/block-editor'
		);
		const selectedBlockClientId = getSelectedBlockClientId();
		return getBlockParents( selectedBlockClientId );
	}, [] );
```